etc/org.opencastproject.adminui.endpoint.LanguageServiceEndpoint.cfg

```diff
@@ -0,0 +1,3 @@
+# Optional param to exclude languages from the admin-UI
+# Add languages to exclude using xx_XX notation, comma separated values
+org.opencastproject.adminui.languages.exclude=
```

Comment:
Can we mention the default and comment out the option if it's set to the default like this:

```
# Optional param to exclude languages from the admin-UI
# Add languages to exclude using xx_XX notation, comma separated values
# Default: <empty>
#org.opencastproject.adminui.languages.exclude=
```